.changeset/ten-rivers-hammer.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: `overrides.files` negated pattern regression
```

Comment:
medium

I would add the version—15.0.0—that introduced the regression.
It's an important information.

Reply:
Good catch. How about this? 👇🏼 

```suggestion
Fixed: `overrides.files` negated pattern regression since 15.0.0
```

Reply:
_since_ sounds fine to me.
_introduced in_ works as well.

Reply:
Thanks. I'll apply your suggestion. 👍🏼 

```suggestion
Fixed: `overrides.files` negated pattern regression introduced in 15.0.0
```